src/ConfigEditor/ConfigSection/GenericConfigSection.tsx

```diff
@@ -0,0 +1,71 @@
+import React, { useState } from 'react';
+import { css } from '@emotion/css';
+import { useTheme2, IconButton, IconName } from '@grafana/ui';
+
+export type Props = {
+  title: string;
+  description?: string;
+  isCollapsible?: boolean;
+  isOpen?: boolean;
+  kind?: 'section' | 'sub-section';
+  className?: string;
+};
+
+export const GenericConfigSection: React.FC<Props> = ({
+  children,
+  title,
+  description,
+  isCollapsible = false,
+  isOpen: isInitiallyOpen = true,
```

Comment:
why just not name it isInitiallyOpen ?

Reply:
Changed to `isInitiallyOpen` in efc89d8228af61123f260f667a106f136b3375b0.